### ISSUE TITLE
Interactive Learning issue

### ISSUE THREAD

**Issue by Akshit99** [RasaHQ/rasa_nlu#1560]
**Rasa NLU version**:
0.12.2
**Operating system** (windows, osx, ...):
windows 7 ultimate
**python version**
3.6.5
**Content of model configuration file**:
```yml

```

**Issue**:
Hi!!! first of all huge fan of the interactive learning process! :) But lately, I'm getting an issue. Whenever i complete my story and try to export to my nlu data files. It removes all the data from my data.json (nlu data files) and creates a new interactive.md file with only those intents i used during training. But when i rerun the model. It says utter_ files missing so i think it can't find those files again in data/data.json
also, when it asks do you want to export nlu data to nlu.md , i change it to data.json which is my data file. Plz help! I love interactive learning and surely don't want to manually write stories. ;P

**Comment by akelad** [comment]
Thanks for raising this issue, @tmbo  will get back to you about it soon.

**Comment by Akshit99** [comment]
so I got a question @akelad @JustinaPetr Whenever I train my bot by using the interactive feature, it's overwriting "action_listen" in my domain file. Then when I run the bot in the console, it says custom action not found "action_listen" gave 404 error. Why is this happening?

**Comment by Akshit99** [comment]
> @Akshit99 I am sure it was an old bug. This should be fixed now if you tried it with the newer version of the Rasa Core.

Yes that was a old bug! I sorted out the issue some time ago! Thanks
